src/stitching/schemaRecreation.ts

```diff
@@ -88,9 +88,10 @@ export function recreateType(
     const newValues = {};
     values.forEach(value => {
       newValues[value.name] = {
-        value: value.name,
+        value: value.value,
```

Comment:
Can I inquire more details about this particular change (`name` to `value`)?

Reply:
Sure thing! Normally when using a `GraphQLEnumType` object, the `name` and `value` properties of an enum value are the same. E.g. `Color.RED` would have `RED` stored as both it's `name` and `value`. `graphql-tools` internal enum value support however needs to store the new internal value we want to use into the new `GraphQLEnumType` value that's created. We have to keep the `name` of the value the same as it was before (e.g. name stays as `RED`), but here we're adjusting the `value` to be the new internal value that's coming from the enum resolver (that we setup in `src/generate/addResolveFunctionsToSchema.ts`).

We could have added some logic in here to only change the `value` to `value.value` when we have enum resolvers setup (in other words only when we want to use new internal enum values), but since `value` is always the same as name when using a normal enum without any internal values, this code change should work in both cases.

Enum with an external value only:
`Color.RED` --> `name` = `RED`, `value` = `'RED'`

Enum with an external and internal value:
`Color.RED` --> `name` = `RED`, `value` = `'#ff0000' `

So using `value.value` in `value` in both cases above gives us what we need.

Reply:
Excellent explanation.  Thank you!